requirements-dev.txt

```diff
@@ -0,0 +1,8 @@
+django-debug-toolbar==1.4 \
```

Comment:
What if we include the `requirements.txt`file on top of this file? That way you could simply update the dev docs to pip install `requirements-dev.txt` instead of `requirements.txt`.

Usually one would put different requirements in the requirements folder, but in case of Heroku we still need to keep the `requirements.txt`, so I think that's not necessary.